llvm/test/CodeGen/AMDGPU/schedule-regpressure-limit-clustering.ll

```diff
@@ -42,4 +42,4 @@ bb2:
 declare i32 @llvm.amdgcn.workitem.id.x() #0
 
 attributes #0 = { nounwind readnone }
-attributes #1 = { "amdgpu-num-vgpr"="9" "amdgpu-flat-work-group-size"="1024,1024" }
+attributes #1 = { "amdgpu-num-vgpr"="9" }
```

Comment:
Unnecessary test change?

Reply:
The load/store interleaving pattern disappears if I leave the attribute (which implies a max. waves/EU of 8, whereas it was 10 before the change or with the default flat workgroup size range).